server/routes/metadata-graph.ts

```diff
@@ -0,0 +1,41 @@
+import * as express from 'express';
+
+const { pgPool } = require('../pool');
+
+const router = express.Router();
+
+router.get('/metadata-graph/:iri', async (req, res) => { 
+  const { iri } = req.params;
+  const iri_re = new RegExp("regen:.+\.rdf");
+  let client;
+  try {
+    if (!iri_re.test(iri)) {
+      res.status(400).send("Invalid IRI, it must of the form regen:<iri-hash>.rdf");
+    } else {
+      client = await pgPool.connect();
+      try {
+         const { rows } = await client.query(
+           'select metadata from metadata_graph where iri=$1',
+           [iri]
+         )
+         if (!rows.length) {
+           res.status(404).send(`metadata_graph with the iri ${iri} not found`)
```

Comment:
We definitely need to add some prettier/eslint to this repo, created an issue for that: https://github.com/regen-network/regen-registry/issues/830

```suggestion
           res.status(404).send(`metadata_graph with the iri ${iri} not found`);
```

Reply:
I'll be glad to pick that one up, it should be a pretty quick task.